common/main/java/com/couchbase/lite/DatabaseChange.java

```diff
@@ -30,13 +30,10 @@
 public class DatabaseChange {
     @NonNull
     private static Collection getDefaultCollection(@NonNull Database database) {
-        CouchbaseLiteException fail = null;
-        try {
-            final Collection defaultCollection = database.getDefaultCollection();
-            if (defaultCollection != null) { return defaultCollection; }
+        try { return database.getDefaultCollection(); }
+        catch (CouchbaseLiteException e) {
+            throw new CouchbaseLiteError("Database " + database.getName() + " has no default collection", e);
```

Comment:
Not sure about the wording here given that the default collection cannot be deleted.  How else would it vanish?

Reply:
Good call.  Fixed